ui/jstests/test-learning-resource.jsx

```diff
@@ -237,7 +239,43 @@ define(['QUnit', 'jquery', 'reactaddons', 'lodash', 'learning_resources',
       };
 
       React.addons.TestUtils.renderIntoDocument(<LearningResourcePanel
+        scrollUpToStatusBox={scrollUpToStatusBox}
+        repoSlug="repo"
+        learningResourceId="1"
+        ref={afterMount} />);
+    }
+  );
+  QUnit.test(
+    'Assert that loader on LearningResourcePanel show and hides properly',
+    function(assert) {
+      var done = assert.async();
+      var scrollUpToStatusBox = function() {};
```

Comment:
Can you keep track of when this function is called and add an assertion that it was called?